### PR TITLE
Enable URL tooltips on hover for Element Desktop

### PR DESCRIPTION
**Pull request by Johennes**
Requires https://github.com/matrix-org/matrix-react-sdk/pull/8394

Closes: #6532

---
Here's what your changelog entry will look like:

## ✨ Features
 * Enable URL tooltips on hover for Element Desktop ([\#22286](https://github.com/vector-im/element-web/pull/22286)). Fixes undefined/element-web#6532. Contributed by @Johennes.

### CODE REVIEW

File: src/vector/platform/ElectronPlatform.tsx [hunk 1]
```diff
@@ -360,6 +360,10 @@ export default class ElectronPlatform extends VectorBasePlatform {
         electron.send('loudNotification');
     }
 
+    needsUrlTooltips(): boolean {
```
Comment (turt2live) [inline src/vector/platform/ElectronPlatform.tsx:363]:
```suggestion
    public needsUrlTooltips(): boolean {
```

bonus points for fixing all the other functions here: we require explicit member access (but the linter isn't intelligent enough to know that yet)

Comment (Johennes) [reply src/vector/platform/ElectronPlatform.tsx]: 👍 Looks like there were only two other methods that had an explicit access level in `BasePlatform` but not here.

**Review by turt2live — APPROVED** [review]

**Comment by turt2live** [comment]
you might need to update your fork's develop and re-run the CI to make it happy.

**Comment by Johennes** [comment]
@turt2live the React SDK PR was merged so I think this can be landed, too, now.

**Comment by Johennes** [comment]
> err, we should really be landing these things at the same time. Will poke at the process for this stuff.

Oh, sorry about that. Should I have explicitly mentioned this PR when asking for the React SDK to be merged?

**Pull request merged**